app/FreshRSS.php

```diff
@@ -90,8 +90,11 @@ private static function initAuth() {
 	}
 
 	private static function initI18n() {
-		Minz_Session::_param('language', FreshRSS_Context::$user_conf->language);
-		Minz_Translate::init(FreshRSS_Context::$user_conf->language);
+		$selectedLanguage = FreshRSS_Auth::hasAccess() ? FreshRSS_Context::$user_conf->language : null;
```

Comment:
Variables should be snake case in this project iirc?

```suggestion
		$selected_language = FreshRSS_Auth::hasAccess() ? FreshRSS_Context::$user_conf->language : null;
```

Reply:
done